<launch>
  <!-- Arguments -->
  <arg name="odom_topic" default="custom_odom" />

  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    <rosparam file="$(find mobile_robot)/cfg/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find mobile_robot)/cfg/costmap_common_params.yaml" command="load" ns="local_costmap"/>    
    <rosparam file="$(find mobile_robot)/cfg/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find mobile_robot)/cfg/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find mobile_robot)/cfg/move_base_params.yaml" command="load" />
    <rosparam file="$(find mobile_robot)/cfg/dwa_local_planner_params.yaml" command="load" />
    <remap from="odom" to="$(arg odom_topic)"/>
  </node>
</launch>
